src/sentry_plugins/slack/plugin.py

```diff
@@ -138,7 +138,7 @@ def get_tag_list(self, name, project):
             return None
         return set(tag.strip().lower() for tag in option.split(","))
 
-    def notify(self, notification):
+    def notify(self, notification, raise_exception=False):
```

Comment:
How come for some of these you use `**kwargs` and here you explicitly have `raise_exception=False`?

Reply:
@MeredithAnya I used `**kwargs` when I didn't actually use the parameter. I can switch it if you think it makes more sense though.

Reply:
@MeredithAnya I believe `**kwargs` is used then you have arbitrary extra fields. See what we have on the base class for notify: https://github.com/getsentry/sentry/blob/master/src/sentry/plugins/base/notifier.py#L9

Reply:
cool that wfm